src/component/InvoiceContainer/index.jsx

```diff
@@ -3,17 +3,32 @@ import Invoice from "../Invoice"
 
 function InvoiceContainer() {
     const [invoices, setInvoices] = useState([])
-
+    const [sort ,setSort] = useState(NaN)
     useEffect(() => {
-        fetch('https://invoicing-api.dev.io-academy.uk/invoices')
+        fetch(`https://invoicing-api.dev.io-academy.uk/invoices?status=${sort}`)
             .then(res => res.json())
             .then((invoiceData) => {
                 setInvoices(invoiceData.data)
             })
-    }, [])
-
+    }, [sort])
+    function statusChange(e){
+        console.log(parseInt(e.target.value))
+        setSort(parseInt(e.target.value))
+    }
     return (
-        <>
+        <><div className="grid grid-cols-2 pb-10 pl-2 lg:inline-block lg:relative lg:-right-[35%]">
+                <div>
+                    <label htmlFor="sort"></label>
+                    <select className="bg-[#F8F9FA] font-semibold text-left" defaultValue="0" name="sort" id="sort" onChange={statusChange} >
+                        <option disabled value="0">Sort by</option>
+                        <option className="" value="1">Paid</option>
+                        <option value="2">Pending</option>
+                        <option value="3">Cancelled</option>
+                        <option value="NaN">Show all</option>
```

Comment:
plz don't use NaN. Just use 0 instead, no status will have that ID and it doesn't make me sad